internal/build/container_updater.go

```diff
@@ -15,17 +15,15 @@ import (
 
 const pauseCmd = "/pause"
 
-type ContainerUpdater interface {
```

Comment:
i guess theoretically one day we might have different kinds of container updaters... this one uses the docker cli, but we might have one that does other stuff?? but idk, i guess we can leave it as as type rather than an interface for now...